app/Services/ProductService.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Services;
+
+use Illuminate\Support\Collection;
+
+class ProductService
+{
+    /**
+     * This function returns total price of given products
+     *
+     * @param Collection $products
+     * @return float
+     */
+    public function getTotalRevenue(Collection $products): float
+    {
+        $revenue =  $products->sum('price');
+        return roundNumber($revenue, 2);
```

Comment:
no need to specify "2" as the second argument if it's the default argument

 return roundNumber($revenue);